radar_control/radar_control.py

```diff
@@ -712,35 +716,36 @@ def radar():
                     time_to_prep_aveperiod = datetime.utcnow() - time_start_of_aveperiod
                     rad_ctrl_print('Time to prep aveperiod: {}'.format(time_to_prep_aveperiod))
 
+                num_sequences = 0
+                time_remains = True
                 while time_remains:
                     for sequence_index, sequence in enumerate(aveperiod.sequences):
 
                         # Alternating sequences if there are multiple in the averaging_period.
                         time_now = datetime.utcnow()
-                        if intt_break:
+                        if intt_break:  # TODO: Check if integration_period_done_time is in past
```

Comment:
For the TODO: I think the next line is doing exactly this so I don't think this TODO is needed?
```suggestion
                        if intt_break:
```

Reply:
oops, I meant to say 'negative' here, was I sleeping? :sleeping: There's a comment above though that asks this question